src/client/content_cao.cpp

```diff
@@ -1865,5 +1884,33 @@ std::string GenericCAO::debugInfoText()
 	return os.str();
 }
 
+void GenericCAO::hideMeshTexture()
```

Comment:
Does not only hide, but also recover visibility. Suggested naming:
`void GenericCAO::updateTextureVisibility()`